### ISSUE TITLE
[BUG] Error Profile Page  v2.4-beta-3

### ISSUE THREAD

**Issue by guarapo-deploy** [Leantime/leantime#1900]
After upgrade the docker image for users/editOwn/ , the users/editOwn page is throwing 500 error

**What set up are you using**
* [ ] Cloud Hosted Version
* [x] Self Hosted

**Describe the bug**

**To Reproduce**
1. Update the version to users/editOwn/
2. Login with a user with an image previously uploaded in older version
3. Go to users/editOwn/
4. 500 error 

**Expected behavior**
page should show the content

**Leantime Version**
Page  v2.4-beta-3

**Server**
Apache

**PHP / MySQL Version**
Docker

**Comment by charly-palencia** [comment]
Related issue https://github.com/Leantime/leantime/issues/1900

**Comment by charly-palencia** [comment]
Here's the error in for the leantime log

```
Next Illuminate\Contracts\Container\BindingResolutionException: Target class [Leantime\Domain\Users\Repositories\Files] does not exist. in /var/www/html/vendor/illuminate/container/Container.php:891
Stack trace:
#0 /var/www/html/vendor/illuminate/container/Container.php(770): Illuminate\Container\Container->build('Leantime\\Domain...')
#1 /var/www/html/vendor/illuminate/container/Container.php(706): Illuminate\Container\Container->resolve('Leantime\\Domain...', Array)
#2 /var/www/html/app/Domain/Users/Repositories/Users.php(645): Illuminate\Container\Container->make('Leantime\\Domain...')
#3 /var/www/html/app/Domain/Users/Controllers/EditOwn.php(82): Leantime\Domain\Users\Repositories\Users->getProfilePicture(1)
#4 /var/www/html/app/Core/Controller.php(91): Leantime\Domain\Users\Controllers\EditOwn->get(Array)
#5 /var/www/html/app/Core/Controller.php(53): Leantime\Core\Controller->executeActions('GET', Array)
#6 [internal function]: Leantime\Core\Controller->__construct(Object(Leantime\Core\IncomingRequest), Object(Leantime\Core\Template), Object(Leantime\Core\Language))
#7 /var/www/html/vendor/illuminate/container/Container.php(929): ReflectionClass->newInstanceArgs(Array)
#8 /var/www/html/vendor/illuminate/container/Container.php(770): Illuminate\Container\Container->build('Leantime\\Domain...')
#9 /var/www/html/vendor/illuminate/container/Container.php(706): Illuminate\Container\Container->resolve('Leantime\\Domain...', Array)
#10 /var/www/html/app/Core/Frontcontroller.php(110): Illuminate\Container\Container->make('Leantime\\Domain...')
#11 /var/www/html/app/Core/Frontcontroller.php(73): Leantime\Core\Frontcontroller::executeAction('users.editOwn', Array, 200)
#12 /var/www/html/app/Core/Bootloader.php(404): Leantime\Core\Frontcontroller::dispatch()
#13 /var/www/html/app/Core/Bootloader.php(160): Leantime\Core\Bootloader->handleRequest()
#14 /var/www/html/public/index.php(14): Leantime\Core\Bootloader->boot()
```

**Comment by steveoriol** [comment]
I confirm, same error...

**Comment by marcelfolaron** [comment]
Fixed in 2.4-beta-4
